### DISCUSSION TITLE
Thirdweb：a web3 application framework

### DISCUSSION THREAD

**Discussion by gofenix** [NonceGeek/Move-Camp#95] [Materials]
https://thirdweb.com/dashboard

thirdweb is a web3 application framework that allows you to build powerful, blockchain-based applications more easily, with tools to use at all layers of your development stack:

Smart Contracts for you to deploy and tools to support building them.
SDKs to interact with the blockchain in your favorite languages.
Dashboards to create, configure, and deploy contracts.

![image](https://user-images.githubusercontent.com/6822558/175194101-013b794d-be45-470d-a59e-15b2abf2834b.png)

**Comment by gofenix** [comment]
参考这里不用写一行合约代码，就可以发一个nft的drop，而且钱包可以去连接测试网络，免费试玩！

<img width="1245" alt="image" src="https://user-images.githubusercontent.com/6822558/175194843-d77b8ce0-fb4c-4fc0-beaa-03e98daa2fb1.png">

**Comment by leeduckgo** [comment]
Great.

**Reply by leeduckgo** [reply]
66R for the recommend.
